setup.py

```diff
@@ -34,8 +34,10 @@
         "Programming Language :: Python :: Implementation :: CPython",
     ],
     python_requires=">=3.8",
-    # TODO: replace "numba-dppy" by "numba-dpex" once released
-    install_requires=["scikit-learn", "numba-dpex", "dpnp"],
+    # TODO: remove pyopencl when dpctl exposes preferred_work_group_size_multiple
+    # see https://github.com/IntelPython/dpctl/issues/886
+    install_requires=["scikit-learn", "numba-dpex", "dpctl", "pyopencl"],
```

Comment:
Just checking: `dpnp` was removed here, but I guess it's not used anymore. Is it the case?

Reply:
indeed